src/btree/bt_vrfy_dsk.c

```diff
@@ -237,6 +237,89 @@ __verify_dsk_value_validity(WT_SESSION_IMPL *session, WT_CELL_UNPACK_KV *unpack,
       cell_num - 1, tag);
 }
 
+/*
+ * __verify_dsk_addr_page_del --
+ *     Verify a deleted-page address cell's page delete information.
+ */
+static int
+__verify_dsk_addr_page_del(WT_SESSION_IMPL *session, WT_CELL_UNPACK_ADDR *unpack, uint32_t cell_num,
+  WT_ADDR *addr, const char *tag)
+{
+    WT_DECL_RET;
+    WT_TIME_AGGREGATE ta_with_delete;
+    char time_string[WT_TIME_STRING_SIZE];
+
+    /* The durable timestamp in the page_delete info should not be before its commit timestamp. */
+    if (unpack->page_del.durable_timestamp > unpack->page_del.timestamp)
```

Comment:
The `durable timestamp` should be equal to or greater than the `commit timestamp`. Is the condition check wrong?

Reply:
Sure looks like it. Not so sure how it passed any tests that way...

Reply:
Fixed in [56c599d](https://github.com/wiredtiger/wiredtiger/pull/7829/commits/56c599de108bafda82587f4e58bb39707cab342f). I added a test, since having 0 test coverage for this clearly isn't good.